spec/timezone.html

```diff
@@ -584,9 +584,9 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-canonicalizetimezoneoffsetstring" type="abstract operation">
+    <emu-clause id="sec-temporal-NormalizeTimeZoneOffsetString" type="abstract operation">
       <h1>
-        CanonicalizeTimeZoneOffsetString (
+        NormalizeTimeZoneOffsetString (
```

Comment:
Are these changes here by mistake?

Reply:
Yeah, mistake. Fixed now. When it happened before I thought I'd overlooked it during the search-n-replace step when reverting the name change. But what actually happened was that rebasing over #2588 un-reverted the change in this PR. Good reminder that the "stacked PR" workflow can be more brittle than you'd naively think that it'd be.  Thanks for catching this.